portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a comment list. */
+public final class Comment {
```

Comment:
Could you comment on why this is not an inner class in the CommentsServlet?

It seems like it could be nested inside the class where it's used, but I might be misunderstanding the reasoning.

Reply:
In the tutorial example(to-do list) they define the Task class in a separate file: 
![image](https://user-images.githubusercontent.com/46226072/83574875-5bf94c80-a4fc-11ea-8026-efac15554d21.png)
It makes more sense in the example since it has more servlets such as delete task, get new task that all use the Task class. 
Since in the future I might implement a delete feature as well, I reused this structure from the example